tool/e2e-tests/helpers.ts

```diff
@@ -286,8 +286,10 @@ async function getCreatedPrescriptionId(driver: ThenableWebDriver): Promise<stri
   return prescriptionId
 }
 
+/* eslint-disable @typescript-eslint/no-unused-vars */
 export async function finaliseWebAction(driver: ThenableWebDriver, log: string): Promise<void> {
-  console.log([log, await driver.takeScreenshot()].join("\n"))
+  //console.log([log, await driver.takeScreenshot()].join("\n"))
```

Comment:
If we want to just get rid of the screenshots all together I reckon that's ok, not sure they are adding much value

Reply:
Leaving in for now, as there's opportunity to help with assurance and I don't want us to lose visibility of this yet

Reply:
+1